README.md

```diff
@@ -1,71 +1,399 @@
+**Step-3:** Upload your **_iOS_** application (.ipa file) or **_android_** application (.apk file) to the LambdaTest servers using our **REST API**. You need to provide your **Username** and **AccessKey** in the format `Username:AccessKey` in the **cURL** command for authentication. Make sure to add the path of the **appFile** in the cURL request. Here is an example cURL request to upload your app using our REST API:
 
-```bash
-curl -u "YOUR_LAMBDATEST_USERNAME":"YOUR_LAMBDATEST_ACCESS_KEY" \
---location --request POST 'https://manual-api.lambdatest.com/app/upload/realDevice' \
---form 'name="Android_App"' \
---form 'appFile=@"/Users/macuser/Downloads/proverbial_android.apk"' 
+**Using App File:**
+
+**Linux/macOS:**
+
+`{`curl -u "${ YOUR_LAMBDATEST_USERNAME()}:${ YOUR_LAMBDATEST_ACCESS_KEY()`}" \\`
+
+`--location --request POST 'https://manual-api.lambdatest.com/app/upload/realDevice' \\`
+
+`--form 'name="Android_App"' \\`
+
+`--form 'appFile=@"/Users/macuser/Downloads/proverbial_android.apk"' 
+`}`
+
+**Windows:**
+
+`{`curl -u "${ YOUR_LAMBDATEST_USERNAME()}:${ YOUR_LAMBDATEST_ACCESS_KEY()}" -X POST "https://manual-api.lambdatest.com/app/upload/realDevice" -F "appFile=@"/Users/macuser/Downloads/proverbial_android.apk""`}`
+
+**Using App URL:**
+
+**Linux/macOS:**
+
+`{`curl -u "${ YOUR_LAMBDATEST_USERNAME()}:${ YOUR_LAMBDATEST_ACCESS_KEY()`}" \\`
+
+`--location --request POST 'https://manual-api.lambdatest.com/app/upload/realDevice' \\`
+
+`--form 'name="Android_App"' \\`
+
+`--form 'url="https://prod-mobile-artefacts.lambdatest.com/assets/docs/proverbial_android.apk"'`}`
+
+**Windows:**
+
+`{`curl -u "${ YOUR_LAMBDATEST_USERNAME()}:${ YOUR_LAMBDATEST_ACCESS_KEY()}" -X POST "https://manual-api.lambdatest.com/app/upload/realDevice" -d "{\"url\":\"https://prod-mobile-artefacts.lambdatest.com/assets/docs/proverbial_android.apk\",\"name\":\"sample.apk\"}"`}`
+
+Tip: 
+
+- If you do not have any **.apk** or **.ipa** file, you can run your sample tests on LambdaTest by using our sample :link: [Android app](https://prod-mobile-artefacts.lambdatest.com/assets/docs/proverbial_android.apk) or sample :link: [iOS app](https://prod-mobile-artefacts.lambdatest.com/assets/docs/proverbial_ios.ipa).
+- Response of above cURL will be a **JSON** object containing the `App URL` of the format - <lt://APP123456789123456789> and will be used in the next step.
+
+## Run Your First Test
+
+### Sample Test With Java
+
+Here are sample JUnit automation scripts used to run on an Android app and iOS app. Ensure to update the `app_url` in the code scripts before running the tests.
+
+<Tabs className="docs__val">
+<TabItem value="android" label="Android" default>
+
+```java title="vanilla_android.java"
+import io.appium.java_client.AppiumDriver;
+import io.appium.java_client.MobileBy;
+import io.appium.java_client.MobileElement;
```

Comment:
why this code is in read me file

Reply:
We have beautified the repos for all frameworks and thus creating an ecosystem where we aim to increase the visibility of LT on Github

Reply:
Please merge these changes as soon as possible.

Reply:
@Devansh013 this appium code is already in the java file why do you want to put it in read me file